pkg/fixtures/writer_blob.go

```diff
@@ -0,0 +1,112 @@
+package fixtures
+
+import (
+	"context"
+	"github.com/applike/gosoline/pkg/blob"
+	"github.com/applike/gosoline/pkg/cfg"
+	"github.com/applike/gosoline/pkg/mon"
+	"github.com/aws/aws-sdk-go/aws"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+)
+
+type BlobFixturesSettings struct {
+	BasePath   string
+	ConfigName string
+}
+
+type blobFixtureWriter struct {
+	logger      mon.Logger
+	batchRunner blob.BatchRunner
+	purger      *blobPurger
+	settings    *BlobFixturesSettings
```

Comment:
again, we don't need the whole settings on the struct if we only get the BasePath from it, do we? Maybe you can also adjust the purger into that direction.

Reply:
Applied.